source/clear-linux/concepts/autospec-about.rst

```diff
@@ -1,112 +1,104 @@
-.. _autospec-about: 
+.. _autospec-about:
 
 Autospec
 ########
 
-.. _incl-autospec-overview:
+``autospec`` is a tool to assist in the automated creation and maintenance of
+RPM packaging in |CL-ATTR|. Where a standard RPM build process using ``rpmbuild``
+requires a tarball and .spec file to start, ``autospec`` requires only a tarball
+and package name to start.
 
-Overview
-********
+How autospec works
+******************
+
+``autospec`` attempts to infer the requirements of the .spec file by analyzing
+the source code and :file:`Makefile` information. It will continuously run
+updated builds based on new information discovered from build failures until it
+has a complete and valid .spec file. Although not required, you can influence
+the behavior of ``autospec`` by providing :ref:`control files <control-files>`.
+
+The basic process is described in the following steps:
+
+#. The :command:`make autospec` command generates a .spec based on
+   analysis of code and control files, if present.
+
+#. ``autospec`` creates a ``build root`` with ``mock`` config.
+
+#. ``autospec`` attempts to build an RPM from the generated .spec.
 
-Whereas a standard RPM build process using ``rpmbuild`` requires a tarball 
-and ``spec`` file to start, ``autospec`` only requires a tarball and package
-name. ``autospec`` analyzes the source code and :file:`Makefile` information 
-in order to generate a ``spec`` file for you. Although not required, you can 
-influence ``autospec`` by providing control files. 
+#. ``autospec`` detects any missed declarations in the .spec.
 
-.. code-block:: console 
+#. If a build error occurs, ``autospec`` stops for user inspection and
+   editing of control files to resolve issues.
```

Comment:
The end goal of autospec is to largely eliminate this step, so autospec will often *automatically* detect the cause of the build failure and restart the build (step 1).

Of course, there are build failures autospec does not detect, and this is where the control files are used, to inform autospec of failure cases it can't detect on its own.

Reply:
@phmccarty. Would you be satisfied with a revision that incorporates your feedback? Please advise: 

Note: However, autospec often detects the cause of the build failure and restarts the build process.

Reply:
@mvincerx I would reword the description. Something like:

_If build errors occur, autospec will scan the build log to try and detect the root cause. If it detects the root cause and knows how to continue, it will restart the build automatically with updated build instructions. Otherwise, it will stop the build for user inspection and editing of control files to resolve the errors._

Reply:
@krisdale. Can you please revise final step with @phmccarty 's recommendation?

Reply:
@mvincerx Revised final step(s). @phmccarty please review to confirm it includes the needed details and info.